serverless.yml

```diff
@@ -0,0 +1,52 @@
+service: aqts-capture-dvstat-transform
+
+provider:
+  name: aws
+  region: ${opt:region, 'us-west-2'}
+  stage: ${opt:stage, 'TEST'}
+  runtime: java8
+  memorySize: 256
+  timeout: 60
+  logRetentionInDays: 30
+  deploymentBucket:
+    name: iow-cloud-applications
+  stackTags:
+    "wma:applicationId": "AQTS Time Series Type Router"
+    "wma:contact": "Jim Kreft jkreft@usgs.gov"
+    "wma:environment": ${self:provider.stage}
+    "wma:taggingVersion": ${opt:taggingVersion}
+    "wma:costCenter": TBD
+
+custom:
+  accountNumber : ${ssm:/iow/aws/accountNumber}
+  vpc:
+    securityGroupIds: ${ssm:/iow/retriever-capture/${self:provider.stage}/securityGroupIds~split}
+    subnetIds: ${ssm:/iow/aws/vpc/${self:provider.stage}/subnetIds~split}
+  db:
+    connectInfo: ${ssm:/aws/reference/secretsmanager/NWCAPTURE-DB-${self:provider.stage}~true}
+
+functions:
+  determineRoute:
+    handler: gov.usgs.wma.waterdata.DetermineRouteHandler
+    role: arn:aws:iam::${self:custom.accountNumber}:role/csr-Lambda-Role
+    environment:
+      FUNCTION_NAME: determineRoute
```

Comment:
These function names might correspond to a different serverless file?

Reply:
Looks like it should be something like transformDailyValue, TransformDailyValueHandler